layers/meta-resin-raspberrypi/recipes-core/enable-overcommit/files/enable-overcommit.conf

```diff
@@ -0,0 +1,2 @@
+vm.overcommit_memory=1
+vm.admin_reserve_kbytes=16000
```

Comment:
Why do we increase this value?

Reply:
For RPI1 the value is small. I found empirically that the system is more responsive with this value rather the default one (~7000)